src/dataExplorer/components/SaveAsScript.tsx

```diff
@@ -111,7 +112,10 @@ const SaveAsScript: FC<Props> = ({language, onClose, setOverlayType, type}) => {
     setResult(null)
     clearViewOptions()
 
-    if (isIoxOrg || isFlagEnabled('influxqlUI')) {
+    // InfluxQL works on both IOx and TSM, so if an account has
```

Comment:
This will also work in OSS, correct? The only different thing I could think of doing differently here is if we need a `CLOUD` check.

Reply:
InfluxQL will not show up in OSS because of [this line](https://github.com/influxdata/ui/pull/6809/files#diff-705df60af76f336059d091ad2442f9d08f2f28ce2ffa6a5c8b947bd84ceef4b0R199), so OSS users will never need to save an InfluxQL script